ddl/ddl_api.go

```diff
@@ -1747,8 +1747,7 @@ func resolveAlterTableSpec(ctx sessionctx.Context, specs []*ast.AlterTableSpec)
 		validSpecs = append(validSpecs, spec)
 	}
 
-	if len(validSpecs) != 1 {
-		// TODO: Hanlde len(validSpecs) == 0.
+	if len(validSpecs) > 1 {
```

Comment:
Will `len(validSpecs) == 0`?

Reply:
Yes, `ALTER TABLE t;` is `len(validSpecs) == 0`.